pyproject.toml

```diff
@@ -24,7 +24,7 @@ aiohttp = "^3.8.1"
 cached-property = "^1.5.2"
 Jinja2 = "^3.0.3"
 jq = "^1.2.2"
-pydantic = "^1.9.1"
+pydantic = "^1.10.1"
```

Comment:
I don't think pydantic should be bumped here if we don't need the update in the lib. Constraints should be as loose as possible for libs, else it can be annoying for the users

Reply:
https://github.com/ToucanToco/toucan-connectors/pull/659/commits/c77f10c8b8686c5eb50205c4b3636ed9ccf97846